webapp/src/utils/utils.tsx

```diff
@@ -11,6 +11,16 @@ function uuid() {
     return uuid.substring(uuid.lastIndexOf('/') + 1);
 }
 
+function dateToUnixTimestamp(date: Date): number {
+    return Math.floor(date.getTime());
```

Comment:
Why do a floor here? I would've thought this would always return an integer because they're Unix timestamps

Reply:
Its not needed now. It was needed previously when I was storing seconds, not milliseconds. Removed now.